tests/auth.test.js

```diff
@@ -0,0 +1,47 @@
+import app from "../app.js";
+import supertest from "supertest";
+import mongoose from "mongoose";
+import "dotenv/config";
+
+describe("auth", () => {
+  beforeAll(async () => {
+    await mongoose.disconnect();
```

Comment:
Are you using real mongodb connect in tests? It's very bad idea. Use mock

Reply:
I'm using a separate mongodb purely for testing, with mock data. Is that a bad idea?

Reply:
Maybe I missed something. What instance of mongodb do you use?

Reply:
I have two db urls defined in my enviroment variable. One is the real database (bugetifydb) that has real data (not currently obviously), and another one that is called bugetifydb-test, that database has mock data used for testing purposes.

Reply:
And in this before all function i connect to it. In 9th line of auth.test.js file.

Reply:
It's better to use in memory db: https://dev.to/paulasantamaria/testing-node-js-mongoose-with-an-in-memory-database-32np